server/server.go

```diff
@@ -125,77 +119,73 @@ func (m *Command) Run(args ...string) (err error) {
 		return fmt.Errorf("server.Open: %v", err)
 	}
 
-	m.Server.Logger.Printf("Listening as %s\n", m.Server.URI)
+	m.logger.Printf("Listening as %s\n", m.Server.URI)
+
 	return nil
 }
 
+// Wait waits for the server to be closed or interrupted.
+func (m *Command) Wait() error {
```

Comment:
the `Run`/`Wait` terminology is odd to me, but I understand what's going on. I also don't have a better suggestion right now.